### PR TITLE
Fix readme links and incorrect mapping of products

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

Slack and demo feedback

## Pull Request Description

Doc updates

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

N/A

### CODE REVIEW

**Review by abowersox-cb — APPROVED** [review]

**Review by lhilmes-cb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -70,11 +70,11 @@ If developing the SDK, you also need:
 
 ## Getting Started
 
-Visit the [Getting Started Guide](https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/getting-started.html) for an example of authenticating and making an API call using the SDK.
+Visit the [Getting Started Guide](https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/getting-started) for an example of authenticating and making an API call using the SDK.
```
Comment (kebringer-cb) [inline README.md:73]: Line 25 says support coming with v1.0.  It's here now.

File: README.md [hunk 2]
```diff
@@ -70,11 +70,11 @@ If developing the SDK, you also need:
...
```
Comment (kebringer-cb) [inline README.md:74]: After the bullet points for products at lines 37-44, and a note that support for newer products and features such as Workload are on the roadmap/coming soon.

Comment (avanbrunt-cb) [reply README.md:75]: Added a note let me know what you think

File: README.md (hunk 1 above)
Comment (avanbrunt-cb) [reply README.md:74]: Modified the sentence so it makes more sense now

**Review by kebringer-cb — APPROVED** [review]
lgtm

**Review by llyon-cb — APPROVED** [review]
LGTM!

**Pull request merged**